### DISCUSSION TITLE
如何控制不在服务器上生成sentinel的文件（csp路径下的那些文件）

### DISCUSSION THREAD

**Discussion by DeBruyne2020** [alibaba/Sentinel#3373] [Q&A]
请教下老师们：
我们使用sentinel只使用了降级，没使用限流，能否控制不在服务器上生成sentinel的那些文件，比如计数文件等等，就是 csp 路径下的那些文件，能控制不生成么？ 前提不影响降级功能（我们用sentinel只用到降级功能，没用到别的）

**Comment by Carlos-Chong200** [comment]
试试 -Dcsp.sentinel.log.dir= -Dcsp.sentinel.metrics.enable=false -Dcsp.sentinel.api.server.port=-1
